From 017f0704f036cde8941fd9194e73f30849c51a92 Mon Sep 17 00:00:00 2001
From: Jerry Jelinek <jerry.jelinek@joyent.com>
Date: Tue, 19 Mar 2019 12:36:52 +0000
Subject: [PATCH] OS-7671 add zpool_sync zfs tests to smartos run file Reviewed
 by: Kody Kantor <kody.kantor@joyent.com> Approved by: Kody Kantor
 <kody.kantor@joyent.com>

---
 usr/src/test/zfs-tests/runfiles/smartos.run | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/usr/src/test/zfs-tests/runfiles/smartos.run b/usr/src/test/zfs-tests/runfiles/smartos.run
index 18a491c403..1155c29025 100644
--- a/usr/src/test/zfs-tests/runfiles/smartos.run
+++ b/usr/src/test/zfs-tests/runfiles/smartos.run
@@ -71,6 +71,9 @@ tests = ['zpool_upgrade_001_pos', 'zpool_upgrade_002_pos',
     'zpool_upgrade_006_neg', 'zpool_upgrade_008_pos',
     'zpool_upgrade_009_neg']
 
+[/opt/zfs-tests/tests/functional/cli_root/zpool_sync]
+tests = ['zpool_sync_001_pos', 'zpool_sync_002_neg']
+
 [/opt/zfs-tests/tests/functional/cli_user/misc]
 tests = ['zdb_001_neg', 'zfs_001_neg', 'zfs_allow_001_neg',
     'zfs_clone_001_neg', 'zfs_create_001_neg', 'zfs_destroy_001_neg',
-- 
2.21.0

